airbyte-cdk/python/unit_tests/base_python/test_schema_helpers.py

```diff
@@ -25,7 +25,7 @@
 import os
 import shutil
 import sys
-from collections import Mapping
+from collections.abc import Mapping
```

Comment:
this was throwing an warning - importing from just `collections` is going to be deprecated with Python 3.9.